test/api/v3/integration/groups/GET-groups_groupId_members.test.js

```diff
@@ -79,15 +79,16 @@ describe('GET /groups/:groupId/members', () => {
 
     expect(memberRes).to.have.all.keys([ // works as: object has all and only these keys
       '_id', 'id', 'preferences', 'profile', 'stats', 'achievements', 'party',
-      'backer', 'contributor', 'auth', 'items',
+      'backer', 'contributor', 'auth', 'items', 'inbox',
     ]);
     expect(Object.keys(memberRes.auth)).to.eql(['timestamps']);
     expect(Object.keys(memberRes.preferences).sort()).to.eql(['size', 'hair', 'skin', 'shirt',
       'chair', 'costume', 'sleep', 'background'].sort());
 
-    expect(memberRes.stats.maxMP).to.exists;
+    expect(memberRes.stats.maxMP).to.exist;
     expect(memberRes.stats.maxHealth).to.equal(common.maxHealth);
     expect(memberRes.stats.toNextLevel).to.equal(common.tnl(memberRes.stats.lvl));
+    expect(memberRes.inbox.OptOut).to.exist;
```

Comment:
We should have an additional test that the contents of inbox.messages does not exist.